content/en/blog/_posts/2025-04-23-Kubernetes-v1-33-Release/index.md

```diff
@@ -0,0 +1,461 @@
+
+This work was done as part of [KEP-4412: Projected service account tokens for Kubelet image credential providers](https://kep.k8s.io/4412) led by SIG Auth.
+
+## Graduations, deprecations, and removals in v1.33
+
+### Graduations to stable
+
+This lists all the features that graduated to stable (also known as *general availability*). For a full list of updates including new features and graduations from alpha to beta, see the release notes.
+
+This release includes a total of 18 enhancements promoted to stable:
+
+* [Take taints/tolerations into consideration when calculating PodTopologySpread skew](https://github.com/kubernetes/enhancements/issues/3094)
+* [Introduce `MatchLabelKeys` to Pod Affinity and Pod Anti Affinity](https://github.com/kubernetes/enhancements/issues/3633)
+* [Bound service account token improvements](https://github.com/kubernetes/enhancements/issues/4193)
+* [Generic data populators](https://github.com/kubernetes/enhancements/issues/1495)
+* [Multiple Service CIDRs](https://github.com/kubernetes/enhancements/issues/1880)
+* [Topology Aware Routing](https://github.com/kubernetes/enhancements/issues/2433)
+* [Portworx file in-tree to CSI driver migration](https://github.com/kubernetes/enhancements/issues/2589)
+* [Always Honor PersistentVolume Reclaim Policy](https://github.com/kubernetes/enhancements/issues/2644)
+* [nftables kube-proxy backend](https://github.com/kubernetes/enhancements/issues/3866)
+* [Deprecate status.nodeInfo.kubeProxyVersion field](https://github.com/kubernetes/enhancements/issues/4004)
+* [Add subresource support to kubectl](https://github.com/kubernetes/enhancements/issues/2590)
+* [Backoff Limit Per Index For Indexed Jobs](https://github.com/kubernetes/enhancements/issues/3850)
+* [Job success/completion policy](https://github.com/kubernetes/enhancements/issues/3998)
+* [Sidecar Containers](https://github.com/kubernetes/enhancements/issues/753)
+* [CRD Validation Ratcheting](https://github.com/kubernetes/enhancements/issues/4008)
+* [node: cpumanager: add options to reject non SMT-aligned workload](https://github.com/kubernetes/enhancements/issues/2625)
+* [Traffic Distribution for Services](https://github.com/kubernetes/enhancements/issues/4444)
+* [Recursive Read-only (RRO) mounts](https://github.com/kubernetes/enhancements/issues/3857)
+
+### Deprecations and removals
+
+As Kubernetes develops and matures, features may be deprecated, removed, or replaced with better ones to improve the project's overall health. See the Kubernetes [deprecation and removal policy](/docs/reference/using-api/deprecation-policy/) for more details on this process. Many of these deprecations and removals were announced in the [Deprecations and Removals blog post](/blog/2025/03/26/kubernetes-v1-33-upcoming-changes/).
```

Comment:
Can you please add the removal of GitRepo in-tree volume driver?
https://github.com/kubernetes/sig-release/discussions/2734#discussioncomment-12603586

Reply:
Thanks for the catch, this should have been caught earlier... I have included a new section with 2fac1ac6ea. 
@xing-yang Would you be able to check if the wording is accurate?

Reply:
Looks good.  Thanks!